### PR TITLE
Update modules.go by adding liquibase-checks extension

### PR DESCRIPTION
**Pull request by rberezen**

### CODE REVIEW

**Comment by rberezen** [comment]
hi @mcred @jnewton03 
would be usefull to add liquibase-checks extension to lpm

**Comment by mcred** [comment]
@rberezen Great idea! This PR also needs the placeholder in the `internal/app/packages.json` file:
```
{
  "name": "liquibase-checks",
  "category": "pro",
  "versions": []
}
```
is likely right. Double check if this is the desired package name.

**Comment by rberezen** [comment]
> @rberezen Great idea! This PR also needs the placeholder in the `internal/app/packages.json` file:
> 
> ```
> {
>   "name": "liquibase-checks",
>   "category": "pro",
>   "versions": []
> }
> ```
> 
> is likely right. Double check if this is the desired package name.

updated pr, name is correct

File: internal/app/packages.json [hunk 1]
```diff
@@ -16674,5 +16674,32 @@
         "liquibaseCore": "4.11.0"
       }
     ]
+  },
+  {
+    "name": "liquibase-checks",
+    "category": "pro",
+    "versions": [
+      {
+        "tag": "1.0.0",
```
Comment (sayaliM0412) [inline internal/app/packages.json:16683]: @mcred : would it cause an issue to have same tag and checksum for different "liquibaseCore" version? I dont think it should but just double checking.

**Comment by mcred** [comment]
fixes #420

**Pull request merged**